src/App.js

```diff
@@ -1,23 +1,61 @@
 import React from "react";
 import "./App.css";
+import Numbers from "./components/ButtonComponents/NumberButtons/Numbers.js";
+import Operators from"./components/ButtonComponents/OperatorButtons/Operators.js";
+import Specials from "./components/ButtonComponents/SpecialButtons/Specials.js";
+import Display  from "./components/DisplayComponents/Display.js";
+
 // STEP 4 - import the button and display components
 // Don't forget to import any extra css/scss files you build into the correct component
 
+
+// import { OperatorButton } from "./components/ButtonComponents/OperatorButtons/OperatorButton.js";
+// import { NumberButton } from "./components/ButtonComponents/NumberButtons/NumberButton.js";
+// import { SpecialButton } from "./components/ButtonComponents/SpecialButtons/SpecialButton.js";
+
+
 // Logo has already been provided for you. Do the same for the remaining components
 import Logo from "./components/DisplayComponents/Logo";
+// import { numberLiteralTypeAnnotation } from "@babel/types";
+
+
 
 function App() {
+  const [displayState, setDisplayState] = React.useState(0);
+  const [numberState, setNumberState] = React.useState();
+  const [operatorState, setOperatorState] = React.useState();
+  const [specialsState, setSpecialsState] = React.useState();
+
   // STEP 5 - After you get the components displaying using the provided data file, write your state hooks here.
```

Comment:
Try to import the hooks as name import to avoid using `React.useState...`

```
import React, { useState } from "react";
```
and the you can 
```
const [displayState, setDisplayState] = useState(0);